docs/pull_request_template.md

```diff
@@ -0,0 +1,9 @@
+Closes #(number) - (issue name)
+
+This PR introduces... (write a simple description of the added feature or solution to the bug)
```

Comment:
Could have headings like the issue templates and might make it easier to quickly glance over a PR